02week/tests.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+if (typeof describe === 'function') {
+
```

Comment:
@LuisCarter8791 
You are missing the test where you check for invalid input, refer to:
https://javascript.austincodingacademy.com/_book/02Week/02DayClass.html#instructions